plutus-benchmark/script-contexts/test/checkScriptContext1.pir-readable.golden

```diff
@@ -96,11 +96,12 @@
                    {unit -> Extended a}
                    (equalsInteger 1 index)
                    (\(ds : unit) ->
-                      let
-                        !arg : data
-                          = headList {data} (sndPair {integer} {list data} tup)
-                      in
-                      Finite {a} (`$dUnsafeFromData` arg))
```

Comment:
Now we know that `Finite` is strict, so `arg` is the first possibly-effectful thing evaluated here, so we can inline it!